airbyte-integrations/connectors/source-bing-ads/source_bing_ads/client.py

```diff
@@ -40,6 +40,8 @@ def __init__(
         customer_id: str,
         client_secret: str,
         client_id: str,
+        tenant_id: str,
```

Comment:
The new parameters should be keyword arguments:
```python

    def __init__(
        self,
        developer_token: str,
        customer_id: str,
        client_secret: str,
        redirect_uri: str,
        refresh_token: str,
        reports_start_date: str,
        hourly_reports: bool,
        daily_reports: bool,
        weekly_reports: bool,
        monthly_reports: bool,
        client_id: str = "common",
        tenant_id: str = "",
        **kwargs: Mapping[str, Any],
    ) -> None:
  ```

Reply:
This change makes the acceptance pass. 
I'm not able to push to because I've no permission to contribute on your organization's Airbyte fork.

Reply:
@alafanechere are you sure? all the rest of connector's params is defined as part of init's signature

Reply:
maybe the better practice is to add the new one to your test config